torchvision/csrc/deform_conv2d.h

```diff
@@ -46,9 +22,9 @@ at::Tensor deform_conv2d_autocast(
     int64_t groups,
     int64_t offset_groups,
     bool use_mask);
-#endif
 
 // C++ Backward
+VISION_API
 std::tuple<at::Tensor, at::Tensor, at::Tensor, at::Tensor, at::Tensor>
 _deform_conv2d_backward(
```

Comment:
nit: do we want to expose `_deform_conv2d_backward` in the headers? I would assume those are private and thus shouldn't be exposed, except if somewhere else in the codebase we need them?

Reply:
I thought to leave it public because we also expose it on Python via the dispatcher. I believe in the past it was exposed to users because it was in a header file, so I opted for including it to avoid BC-breaking changes. Happy to remove it if we think it's not needed.

Reply:
The fact that we have the `*_backward` functions exposed in Python is an artifact of how some things currently work in the dispatcher IIUC. We don't expose `deform_conv2d_backward` under `torchvision.ops`, so I would be ok removing them from those headers.

Reply:
Done at #3143